TLI-Project/Woozles
# Integrate front end to graph 115 sophia

## Context
---

This PR integrates the graphing algorithm with the newly implemented passing states by returning a different page depending on how many models are being compared. **This is operating under the assumption that carList[0].depreciation for example is a list type.** Changes have also been made to the amortization graphing algorithm to accommodate the data structure. 

Fixes SE-115

### Description

- Develops a new page with graphs depending on how many models have been chosen to compare.

### Type of change

Please select options that are relevant.


## How Can This Be Tested?
---

Since the data isn't set up yet, it's not really possible to test right now. A good way to review would be to look at the structure of the data being assigned to the different models eg. modelA (which is the model with data for the amortization graph), modelD (which is the model with data for the depreciation graph) and making sure they align with how the data will look when it's being passed. 

A brief revision of the amortization and depreciation graph to make sure the data is being handled properly would also be really helpful.

## Notes:
---

## Checklist:
---

- [ ] I have given testing instructions for the reviewer 
- [X] Jira ticket number in PR title
